test/perf/perfutil.jl

```diff
@@ -77,7 +81,7 @@ macro timeit(ex,name,desc,group...)
             end
             i += 1
         end
-        @output_timings t $name $desc $group
+        @output_timings t $(esc(name)) $(esc(desc)) $(esc(group))
```

Comment:
Escapes are stripped out as part of expanding a single macro. (There are also 2 macros here, which you might only be able to see by expanding the diff.)